pyproject.toml

```diff
@@ -15,11 +15,29 @@ build-backend = "setuptools.build_meta:__legacy__"
 
 
 [tool.black]
-# Uncomment if pyproject.toml worked fine to ensure consistency with flake8
-# line-length = 120
+line-length = 88
```

Comment:
I don't follow why this line change length is necessary?

Reply:
This change has no effect because 88 is the default line length of `black`. I think explicit things are better than implicit.